applications/dir/prod/values.yaml

```
@@ -0,0 +1,274 @@
+# Ref: https://github.com/agntcy/dir/blob/main/install/charts/dir/values.yaml
+
+apiserver:
+  log_level: INFO
+
+  image:
+    repository: ghcr.io/agntcy/dir-apiserver
+    tag: latest
+    pullPolicy: IfNotPresent
+    pullSecrets: []
+
+  # Expose the P2P API via NodePort for external access
+  routingService:
+    type: NodePort
+    nodePort: 30555
+    # Preserve source IPs for P2P (recommended)
+    externalTrafficPolicy: Local
+
+  # Expose the DIR API via ClusterIP for internal access
+  service:
+    type: ClusterIP
+
```

Or generate another one with `ssh-keygen -t ed25519 -f node.privkey -N ""`. This hardcoded key wrks, because the dir helm chart disable autoscaling by default and only 1 replica is deployed which is appropriate for bootstrap node.

```suggestion
  # Expose the DIR API via ClusterIP for internal access
  service:
    type: ClusterIP

  secrets:
    # Node identity private key
    privKey: |
      -----BEGIN OPENSSH PRIVATE KEY-----
      b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAMwAAAAtzc2gtZW
      QyNTUxOQAAACAw0p3UuQnUpF6n3PO7n8Bg0Bcsy4LN0/XZlGu68RhogQAAAKDt3YP87d2D
      /AAAAAtzc2gtZWQyNTUxOQAAACAw0p3UuQnUpF6n3PO7n8Bg0Bcsy4LN0/XZlGu68RhogQ
      AAAEACPHtDTusTL25Mpenn/WVHCuaMH6XhEfN+CTHq5XHxtDDSndS5CdSkXqfc87ufwGDQ
      FyzLgs3T9dmUa7rxGGiBAAAAFnRraXJjc2lAVEtJUkNTSS1NLUtKV1gBAgMEBQYH
      -----END OPENSSH PRIVATE KEY-----```

---

Ok. Use precreated k8s secret. Then we do not need this `secrets` tag.